spec/topological_inventory/core/service_plan_client_spec.rb

```diff
@@ -4,18 +4,25 @@ module TopologicalInventory
   module Core
     describe ServicePlanClient do
       describe "#build_payload" do
+        let(:order_parameters) do
+          {
+            :service_parameters => service_parameters,
+            :provider_control_parameters => {:namespace => "namespace"}
+          }
+        end
+        let(:service_parameters) { "service_parameters" }
```

Comment:
@eclarizio This should be a hash, it would help someone looking at the code. The service parameter is specific to a service that you are trying to provision.